README.md

```diff
@@ -25,6 +25,42 @@ java -jar cryptomator-cli-x.y.z.jar \
 # you can now mount http://localhost:8080/demoVault/
 ```
 
+## Using as a docker image
+
+*Bridge networking with port forward:*
+
+### !!! WARNING: This approach should only be used to test the containerized approach, not in production.
```

Comment:
Just a little markdown semantics improvement. Also, use emoji to highlight the warning

```suggestion
### Bridge networking with port forward:

:warning: **WARNING: This approach should only be used to test the containerized approach, not in production.** :warning:
```